### PR TITLE
First commit of new Manual EDL Change pack

### PR DESCRIPTION
**Pull request by ali-sawyer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
N/A

## Description
Add new pack that handles the following use case:

Simple, manual process to modify external dynamic lists (EDLs) in XSOAR. This pack may help replace an existing manual process for updating firewall allowlists and blocklists, so analysts may make these changes directly in XSOAR. Simply paste in a list of indicators to add or remove them from the EDL. 

## Screenshots
Playbook screenshot is in README.

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0
- [x] 6.6.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by bakatzir** [comment]
@ali-sawyer Hey, let's connect, as we usually refrain from releasing community packs for PANW products.

**Comment by bakatzir** [comment]
Let's see what we need to do in order to add this PB to the [Generic Export Service pack](https://xsoar.pan.dev/marketplace/details/EDL). as this is a useful flow.

**Comment by ali-sawyer** [comment]
@bakatzir sure, I have a meeting scheduled with @yucohen and @tomer-pan on April 4, would you like to be added to that or should we meet before then?

File: Packs/EDLChange/IncidentTypes/incidenttype-EDL_Change.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "daysR": 0,
+    "weeksR": 0,
+    "system": false,
+    "readonly": false,
+    "default": false,
+    "autorun": true,
+    "disabled": false,
+    "reputationCalc": 0,
+    "onChangeRepAlg": 0,
+    "layout": "d67303c1-bff3-45f3-82ae-024bc61b5f92",
```
Comment (tomer-pan) [inline Packs/EDLChange/IncidentTypes/incidenttype-EDL_Change.json:23]: @ali-sawyer Please change to the correct value (should be the layout name).

Comment (ali-sawyer) [reply Packs/EDLChange/IncidentTypes/incidenttype-EDL_Change.json]: Done!

**Review by tomer-pan — APPROVED** [review]

**Comment by ShahafBenYakir** [comment]
@yucohen @tomer-pan what does this waits for?

**Comment by yucohen** [comment]
@ShahafBenYakir I'll be moving these changes to the [Generic Export Service pack](https://xsoar.pan.dev/marketplace/details/EDL) today

**Comment by ShahafBenYakir** [comment]
@yucohen nice! please sync with @moishce

**Comment by yucohen** [comment]
merging to internal, the validation should not fail once the new sdk version is released

**Pull request merged**